internal/unix/types_linux.go

```diff
@@ -1,3 +1,4 @@
+//go:build linux
```

Comment:
This is 1.17+ syntax, while this module targets 1.16+

Reply:
that's a magic trick from my editor
corrected thanks